### PR TITLE
Minio support to substitute s3 on dev

### PR DESCRIPTION
**Pull request by dLobatog**
Continues from https://github.com/RedHatInsights/insights-upload/pull/39 - that fork repo was borked when this repo went public 

This commit adds a minio.io container to docker-compose. This container
has the same API as s3 for writing/reading/etc, and it also runs on port
9000 of the Docker host.

Therefore this allows Insights app developers to use insights-upload
with this S3-compatible backend, instead of using S3 accounts (which
cost $$$), or localhost - which doesn't work for developing insights
apps as the consumers will need to fetch the file from a S3 bucket, not
from a file in a container.

### CODE REVIEW

**Comment by dLobatog** [comment]
This update contains now things like:

* A .env file for S3 keys and bucket folder locations.
* A new entry in the docker-compose file which creates the buckets
* SELinux handling (using :Z when creating the buckets)
* Removed kafka minio dependency
* Updated README

**Comment by dLobatog** [comment]
@SteveHNH I should update that part of the README - I wrote it before I added the container to docker-compose that actually creates the buckets.

**Comment by dLobatog** [comment]
Updated :smile:

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -5,6 +5,32 @@ services:
...
+    command: server /data
+    volumes:
+      - '$MINIO_DATA_DIR:/data:Z'
+      - '$MINIO_CONFIG_DIR:/root/.minio:Z'
+    ports:
+      - 9000:9000
+    environment:
+      - MINIO_ACCESS_KEY=$MINIO_ACCESS_KEY
+      - MINIO_SECRET_KEY=$MINIO_SECRET_KEY
+  createbuckets:
```
[the remark this answers is not in the document]
Comment (akofink) [reply docker/docker-compose.yml]: Yes, it's idempotent. Why is this scary? It's only used for development. It's basically just making some directories in `/mnt/data` if they don't exist already.

**Pull request merged**